docs/README.md

```diff
@@ -30,7 +30,7 @@ make html-noplot
 
 Shortcut to build the docs (with env variables for active-learning tutorial) use:
 ```
-LIGHTLY_SERVER_LOCATION='https://api.lightly.ai' TOKEN='YOUR_TOKEN' AL_TUTORIAL_DATASET_ID='YOUR_DATASET_ID' make html && python -m http.server 1234 -d build/html
+LIGHTLY_SERVER_LOCATION='https://api.lightly.ai' LIGHTLY_TOKEN='YOUR_TOKEN' AL_TUTORIAL_DATASET_ID='YOUR_DATASET_ID' make html && python -m http.server 1234 -d build/html
```

Comment:
Some of the tutorials make actual API calls. What happens is that `make html` runs the python scripts of the tutorials to create the html output. But the change should not break anything. Not sure why the prod api is used there, maybe @shaundaley39 knows more?

Reply:
In `lightly/api/api_workflow_client.py`, we already reference LIGHTLY_TOKEN, so this change is absolutely good and necessary.

R.e. why/ why not prod...

I do not remember a particular reason to use the prod API when running tutorials to build docs!

Scenario: we have some new unreleased feature in the API that we want to showcase with a self supervised learning package tutorial. We write and run the tutorial using the staging API. We want to be able to build the docs, including this new tutorial, so we can ship the new docs within seconds of deploying the new API feature. This requires us to build the docs using the staging API.

For situations like that, we would want to use the staging API.

Otherwise, there is a small advantage in using the prod API: builds and releases of the pip package and pip docs can be completely decoupled from platform deployments (not dependent on the stage deployment currently being in a good state, etc).